packages/ckeditor5-dev-utils/lib/bundler/geteditorconfig.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Transforms specified configuration to a string which can be inserted
+ * to generated an entry file.
+ *
+ * Keys and values will be wrapped in apostrophes instead of quotation marks.
+ *
+ * Keys which contains letters or/and numbers or/and underscore or/and a Dollar sign
+ * will not be wrapped in apostrophes.
+ *
+ * @param {Object} config
+ * @returns {String}
+ */
+module.exports = function getEditorConfig( config ) {
```

Comment:
This isn't very safe, unfortunately. E.g. every `"` character will be replaced with `'` which certainly isn't correct.

Therefore, let's do the simplest possible thing now – output a JSON **string** to the entry file and parse it there. The entry file should look like this:

```
const config = JSON.parse( '...' );
```

Or, to keep the config more readable in the entry file:

```
const config = JSON.parse(
`{
    "foo": "bar"
}`
);

Ooor... we can keep the editor config in a separate file (`editor-config.js`) – next to `ckeditor.js` and simply import it. Then, we won't need to stringify anything. You could check this option first.